packages/polyfill/source/Element.ts

```diff
@@ -64,6 +64,10 @@ export class Element extends ParentNode {
     return sib;
   }
 
+  constructor(name: string = '') {
```

Comment:
Why do we need this fallback?

Reply:
I kept the behaviour that existed before. I can remove it and enforce the element name as a requirement.